components/Avatar/avatar.css

```diff
@@ -0,0 +1,38 @@
+@import url("/style.css");
+
+.container-avatars {

```

Comment:
you can remove and use common class for every container as said in alert pr.

Reply:
Done